servers/rendering/rendering_device.cpp

```diff
@@ -181,13 +181,34 @@ Ref<RDShaderSPIRV> RenderingDevice::_shader_compile_spirv_from_source(const Ref<
 
 	Ref<RDShaderSPIRV> bytecode;
 	bytecode.instantiate();
-	for (int i = 0; i < RD::SHADER_STAGE_MAX; i++) {
-		String error;
+	String error;
+	ShaderStage stage;
+	Vector<uint8_t> spirv;
+
+	if (p_source->get_stage_source(RD::SHADER_STAGE_COMPUTE) != String()) {
+		for (int i = 0; i < RD::SHADER_STAGE_MAX; i++) {
+			ShaderStage currentStage = ShaderStage(i);
```

Comment:
We use `snake_case` for identifiers:
```suggestion
			ShaderStage current_stage = ShaderStage(i);
```